linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_loghost.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora
```

Comment:
https://devops.profitbricks.com/tutorials/configure-remote-logging-with-rsyslog/

Reply:
SUSE: https://forums.suse.com/showthread.php?6801-Setting-up-sending-Linux-logs-to-central-Syslog-server